packages/blade/src/components/AutoComplete/_decisions/decisions.md

```diff
@@ -0,0 +1,257 @@
+# AutoComplete
+
+A SelectInput where you can type inside the input to filter through the items.
+
+<img width="215" alt="image" src="https://github.com/razorpay/blade/assets/30949385/63d95ab7-6231-4986-befb-1634269e7af3">
+
+## Design
+
+- [Figma - SelectInput & AutoComplete](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=13590%3A171038&mode=dev)
+
+## Props
+
+> **Note**
+>
+> Below this section, I have added [examples of common AutoComplete usage](#usage). Check them out to understand usage of these props in details.
+
+| **Props**          | **Description**                                                                                                                             | **Type**                                                        | **Default Value** |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------- | ----------------- |
+| label              | label of input                                                                                                                              | string                                                          |                   |
+| accessibilityLabel | aria-label of input when label is not defined                                                                                               | string                                                          |                   |
```

Comment:
Hopefully we have an internal mechanism to mandate the `accessibilityLabel` when `label` is not defined? If yes, this should be mentioned in the props table.

Reply:
Yup its consistent for all inputs. Will add in the description as well.

Reply:
done